surveys/models.py

```diff
@@ -78,9 +84,7 @@ def get_observations_from_component(self, component_name):
                 )
         else:
             raise CensusObservation.DoesNotExist(
-                'No SurveyComponent object found with the name: {}'.format(
-                    component_name
-                )
+                'No SurveyComponent found with the name: {}'.format(component_name)
```

Comment:
Looks like we'll need a new migration to reflect this. When I run `python manage.py migrate` I get this:

```
  Your models have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
```

Reply:
I think the missing migration is actually due to the `SurveyChart.census_areas` attribute. Oops! I'll add it in.